meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor/balena-supervisor.service

```diff
@@ -20,13 +20,14 @@ Type=simple
 Restart=always
 RestartSec=10s
 WatchdogSec=180
-SyslogIdentifier=resin-supervisor
-EnvironmentFile=/etc/resin-supervisor/supervisor.conf
+SyslogIdentifier=balena-supervisor
+EnvironmentFile=/etc/balena-supervisor/supervisor.conf
 EnvironmentFile=-/tmp/update-supervisor.conf
-ExecStartPre=-@BINDIR@/balena stop resin_supervisor
+ExecStartPre=-@BINDIR@/balena stop balena_supervisor
 ExecStartPre=/bin/systemctl is-active balena.service
-ExecStart=/usr/bin/healthdog --healthcheck=/usr/lib/resin-supervisor/resin-supervisor-healthcheck @BINDIR@/start-resin-supervisor
-ExecStop=-@BINDIR@/balena stop resin_supervisor
+ExecStart=/usr/bin/healthdog --healthcheck=/usr/lib/balena-supervisor/balena-supervisor-healthcheck @BINDIR@/start-balena-supervisor
+ExecStop=-@BINDIR@/balena stop balena_supervisor
 
 [Install]
 WantedBy=multi-user.target
+Alias=resin-supervisor.service
```

Comment:
Note that the commit Add symlinks and aliases for legacy resin namespaces is a placeholder and may be removed after a product call discussion and replaced with a major version bump.
https://jel.ly.fish/bed172f4-ebd0-4189-95c9-32235a1ea3de